enforcer-rules/src/it/return-type-mismatch/verify.groovy

```diff
@@ -0,0 +1,16 @@
+def buildLog = new File(basedir, "build.log").text.replaceAll("\\r\\n", "\n")
+
+// protobuf-java 3.12.4 has wrong reference to ByteBuffer's methods that are unavailable in Java 8.
+// In this message below, the expectation is wrong.
+assert buildLog.contains('''\
+java.nio.ByteBuffer's method position(int) is expected to return java.nio.ByteBuffer\
+ but the actual type is java.nio.Buffer;
```

Comment:
This is the effect of this PR.

(The backslash at the end is not to break the line there.)